docs/source/tutorials/oss.rst

```diff
@@ -104,5 +104,45 @@ The above `train` function will then need to be run via a `multiprocessing.spawn
         )
 
 
-to see it in action, you can test it with the following script `here <../../../examples/tutorial_oss.py>`_. 
+to see it in action, you can test it with the following script `here <../../../examples/tutorial_oss.py>`_.
 
+Using PyTorch Automatic Mixed Precision is possible, but it requires a shard-aware GradScaler, which is available in
+`fairscale.optim.grad_scaler`. Autocast can be used as is, and the loss will be scaled and handled in the same way.
+See [the original documentation] (https://pytorch.org/docs/stable/notes/amp_examples.html?highlight=automatic%20mixed%20precision)
+for more information.
+
+.. code-block:: python
+
+
+
```

Comment:
why 3 blank lines?

Reply:
it popped up when `make`-ing the doc if I remember correctly, I can check that again